Formula/pixie.rb

```diff
@@ -0,0 +1,31 @@
+class Pixie < Formula
+  desc "Observability tool for Kubernetes applications"
+  homepage "https://px.dev/"
+  url "https://github.com/pixie-io/pixie.git",
+      tag:      "release/cli/v0.7.14",
+      revision: "c15a89eebe0849f16b83ac369cd21d2e6911b1a2"
+  license "Apache-2.0"
+  head "https://github.com/pixie-io/pixie.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    semver = build.head? ? "0.0.0-dev" : version
+    ldflags = %W[
+      -X px.dev/pixie/src/shared/goversion.buildSCMRevision=#{Utils.git_short_head}
+      -X px.dev/pixie/src/shared/goversion.buildSCMStatus=Homebrew
```

Comment:
This will also indicate if people made their own version of this formula.
```suggestion
      -X px.dev/pixie/src/shared/goversion.buildSCMStatus=#{tap.user}
```